File: contracts/startup/startup.py
print("Hello, from the [startup] script!")
def ExecuteEndpoint():
    print(SmCtx.SrcFlow.lower())
    if SmCtx.SrcFlow and SmCtx.SrcFlow.lower() == "request-otp":
        SmCtx.SetResult({
            "Command":"Visit",
            "EndpointUrl":"/api/subscriptions/sub-startup/visit/default/verify-otp",
            "SubmitUrl":"/api/subscriptions/sub-startup/contracts/default/verify-otp",
        })
    elif SmCtx.SrcFlow and SmCtx.SrcFlow.lower() == "post-create-bankaccount":
        SmCtx.SetResult({
            "Command":"Visit",
            "EndpointUrl":"/api/subscriptions/sub-startup/visit/default/verify-otp",
        })
    elif SmCtx.SrcFlow and SmCtx.SrcFlow.lower() == "verify-otp":
        SmCtx.SetResult({
            "Command":"Visit",
            "EndpointUrl":"/api/subscriptions/sub-startup/visit/default/account-display",
            "SubmitUrl":"/api/subscriptions/sub-startup/contracts/default/account-display",
        })
    elif SmCtx.SrcFlow and SmCtx.SrcFlow.lower() == "account-display":
        SmCtx.SetResult({
            "Command":"Visit",
            "EndpointUrl":"/api/subscriptions/sub-startup/visit/default/pin",
            "SubmitUrl":"/api/subscriptions/sub-startup/contracts/default/pin",
        })
    elif SmCtx.SrcFlow and SmCtx.SrcFlow.lower() == "pin":
        SmCtx.SetResult({
            "Command":"Visit",
            "EndpointUrl":"/api/subscriptions/sub-startup/visit/default/login-result",
            
            "LoginResult":"success",
        })

File: contracts/lua/startup.lua
-- local json = require 'Services\\LuaExecution\\json'
print("Hello, from the [startup] script!")
function ExecuteEndpoint()
    print(SmCtx.SrcFlow)
    if (string.lower(SmCtx.SrcFlow) == "try-lua" and string.lower(SmCtx.CallType) == "visit") then
        SmCtx:SetResult({
            Command = "Visit",
            EndpointUrl = "/api/subscriptions/topup/visit/default/topuplist?EndpointId=" .. SmCtx.SrcEndpointId
        })
    elseif (string.lower(SmCtx.SrcFlow) == "try-lua" and string.lower(SmCtx.CallType) == "get-data") then
        feeds = SmCtx:List("BizAccount")
        -- feeds = SmCtx:Get("BizAccount","00001")
        response = {
            name = 'im LUA',
            age = 22,
            status = 'die',
            feeds = feeds,
            testarray = {{
                aaa = "123",
                bbb = 456
            }, {
                aaa = "123",
                bbb = 456
            }}
        }
        SmCtx:SetResponse(feeds)
    elseif (string.lower(SmCtx.SrcFlow) == "try-lua" and string.lower(SmCtx.CallType) == "submit-data") then
        SmCtx:SetResult({
            Command = "Visit",
            EndpointUrl = "/api/subscriptions/topup/visit/default/topuplist?EndpointId=" .. SmCtx.SrcEndpointId
        })
    end
end
